src/debug_menu.cpp

```diff
@@ -956,8 +956,24 @@ void character_edit_menu()
             }
         }
         break;
-        case D_FLU: {
-            p.add_effect( effect_flu, 1000_minutes );
+        case D_ADD_EFFECT: {
+            const auto text = string_input_popup()
```

Comment:
Would be better if it actually lists all effect ids using ui_list.

Reply:
I just looked into that and it looked harder than I would have thought.  Effects aren't a generic factory and while I'm sure I could hack something together, it feels like it needs a bigger effort than this PR to do right.  For now this is only debug so I think it can be a little rough.